test/testenv/testenv.go

```diff
@@ -124,6 +125,18 @@ const (
 
 	// CRUpdateRetryCount if CR Update fails retry these many time
 	CRUpdateRetryCount = 10
+
+	// ProbeConfigMapTemplateStr Config map for liveness and redinesss probe
+	ProbeConfigMapTemplateStr = "splunk-%s-probe-configmap"
+
+	// LivenessScriptName Name for liveness script
+	LivenessScriptName = "livenessProbe.sh"
+
+	// ReadinessScriptName Name for Readiness script
+	ReadinessScriptName = "readinessProbe.sh"
+
+	// ProbeMountDirectory location for liveness and redinesss scripts
+	ProbeMountDirectory = "/mnt/probes"
```

Comment:
Is it needed to redefine the values here? There are getters for all these values in the enterprise package.